### ISSUE TITLE
Cordova plugin google maps master- Plugin not defined

### ISSUE THREAD

**Issue by hornok** [mapsplugin/cordova-plugin-googlemaps#818]
Hey guys I am using Cordova plugin google maps master since 21-jan-2015 morning. I am unable to display a simple google map using this plug in. It says "Uncaught ReferenceError: plugin is not defined". I am using this plugin in visual studio 2015. Please help me out! I am a beginner.

If you guys already have a sample of how to build a simple google map using cordova plugin please share it with me.
 Thank you!
Sidharth.

**Comment by FoCoPeazy** [comment]
I'm getting the same error in iOS with the latest version of the plugin, v1.3.9. A few more details about the iOS setup:

*Initialization is waiting until the 'deviceready' is fired as indicated in the docs and confirmed by #164 . This is my exact code:
```javascript
document.addEventListener('deviceready', function() {
	self.map = plugin.google.maps.Map.getMap(self.mapCanvas, {
		//init options here
	});
});
```
*The plugin is correctly installed. Output of 'cordova plugin list':
```
com.googlemaps.ios 1.11.1 "Google Map iOS SDK for Cordova"
...some others...
cordova-plugin-geolocation 2.1.0 "Geolocation"
...some others...
plugin.google.maps 1.3.9 "phonegap-googlemaps-plugin"
```

*Output of 'cordova --version' is 5.4.1. I have tried removing/re-adding the iOS platform using multiple versions of Cordova-iOS including 3.6.3, 3.8.0, and 3.9.2. I have tried removing/re-adding the plugin at various stages, and it fails 100% of the time regardless of when the plugin is added or what the platform version is.

*The 'Deployment Target' in Xcode is set to 7.0, 'Devices' is set to 'Universal'. I am using Xcode 7.2.

*The problem occurs while using the simulator, as I unfortunately don't have access to a real device. The simulator is running iOS 9. I have not tried other versions of iOS yet.

**Comment by hornok** [comment]
Hello FoCoPeazy,

Thank you very much for your reply, I have understood the reason for the occurrence of the error. 

With regards,
Sidharth.

**Comment by FoCoPeazy** [comment]
@hornok Glad to hear it! Was it an issue of not waiting for the 'deviceready' event?

In regards to iOS, I just now created a new cordova project and followed the example provided in the docs and was able to see the map no problem. One difference I noticed immediately with the new example project that *does* work was that in Xcode there were 22 'Compile Sources' under 'Build Phases'. More specifically, all of the source files under Plugins/plugin.google.maps were there. However, with my existing project that has not been the case--no matter the order in which I add the plugin/iOS platform, I have not yet seen any of these files listed under 'Compile Sources'.

My issue then is to figure out *why* that is the case.
